example/example-model/src/main/java/uk/gov/gchq/palisade/example/rule/BankDetailsRule.java

```diff
@@ -35,6 +35,8 @@ public BankDetailsRule() {
     private Employee redactRecord(final Employee redactedRecord) {
         redactedRecord.setBankDetails(null);
         redactedRecord.setTaxCode(null);
+        redactedRecord.setSalaryAmount(-1);
```

Comment:
Would zero be a better choice here, -1 could screw up follow on processing in a real world example.

Reply:
possibly, my concern was that 0 could be a real value (working for free), but nobody pays a company to work for them. a better approach might be to set the type to Integer rather than a primitive int. That way we could set it to null

Reply:
Yeh, this would make sense too. No approach is superior here to my mind. We can go either way.